modules/diskq/dqtool.c

```diff
@@ -506,6 +520,50 @@ dqtool_relocate(int argc, char *argv[])
   return 0;
 }
 
+static gboolean
+_assign_validate_options(void)
+{
+  return _validate_persist_file_path(persist_file_path);
```

Comment:
In the previous PR you wrote a nice `_validate_diskq_path` that could be used here too.

Reply:
@furiel : not really, as that function is for validating directories, not files

Reply:
Indeed, I wrote wrong function. `_file_is_diskq` is the one that can validate that a correct diskq is passed as parameter. As discussed irl, this can be useful thing to validate.